commands/getStats.js

```diff
@@ -0,0 +1,233 @@
+const sqlite = require('sqlite');
+const sqlite3 = require('sqlite3');
+const AsciiTable = require('ascii-table');
+const config = require('../config');
+
+module.exports = {
+  // get number of verified users
+  // linked to STATS command
+  "getNumVerifiedStats" : async function () {
+    // open db
+    let db = await sqlite.open({
+      filename: config.db_path,
+      driver: sqlite3.Database,
+    });
+  
+    // get count of users in table
+    let row = null;
+    try {
+      row = await db.get(
+        `
+  SELECT
+    count(*) 'count'
+  FROM users`
+      );
+    } catch (e) {
+      console.error(e.toString());
+      await db.close();
+      return [{ message: e.toString() }, null];
+    }
+    if (!row) {
+      return [
+        null,
+        `
+  No stats available on verified users.`,
+      ];
+    }
+  
+    return [
+      null,
+      `
+  \`\`\`
+  Number of Verified Users: ${row.count}
+  \`\`\`
+      `
+    ];
+  },
+
+  // get count of each major
+  // linked to STATS command
+  "getMajorStats" : async function () {
+    // open db
+    let db = await sqlite.open({
+      filename: config.db_path,
+      driver: sqlite3.Database,
+    });
+  
+    // get count of each major
+    let rows = null;
+    try {
+      rows = await db.all(
+        `
+  SELECT
+    major, count(*) 'count'
+  FROM users
+  GROUP BY major
+  ORDER BY count DESC`
+      );
+    } catch (e) {
+      console.error(e.toString());
+      await db.close();
+      return [{ message: e.toString() }, null];
+    }
+    if (!rows) {
+      return [
+        null,
+        `
+  No stats available on majors.`,
+      ];
+    }
+  
+    let majorTable = new AsciiTable('Majors by Count');
+    majorTable.setHeading('Major','Count');
+    rows.forEach(row => majorTable.addRow(row.major, row.count));
+  
+    return [
+      null,
+      `
+  \`\`\`
+  ${majorTable.toString()}
+  \`\`\`
+      `
+    ];
+  },
+
+
```

Comment:
just being nitpicky haha but let's avoid unnecessary newlines! Mostly for consistency imo. I think having a single newline between functions is sufficient

Reply:
i won't comment on every extraneous newline so make sure to go over all the code!